lib/node_modules/@stdlib/math/base/special/fresnel/README.md

```diff
@@ -0,0 +1,76 @@
+# fresnel
+
+> Compute the [Fresnel integral][fresnel-integral].
```

Comment:
Wording: "Compute both parts of the Frensel integral"? "Both components"? "S(x) and C(x)"?

Reply:
> Compute the Fresnel integrals S(x) and C(x).

Reply:
Ah, I was overcomplicating the wording since it's a single integral if you're treating it as complex. Plural is better. 👍